Objects/object.c

```diff
@@ -1830,6 +1830,7 @@ _PyTypes_Init(void)
     INIT_TYPE(&PyMethodDescr_Type, "method descr");
     INIT_TYPE(&PyCallIter_Type, "call iter");
     INIT_TYPE(&PySeqIter_Type, "sequence iterator");
+    INIT_TYPE(&PyPickleBuffer_Type, "pickle.PickleBuffer");
```

Comment:
Maybe it can be only initialized once _pickle is imported? It's only exposed by the _pickle module, no?

Reply:
The goal is for third-party C code to be able to instantiate it directly.

Reply:
Oh, I didn't realize that it should/can be used by 3rd party C extension. In that case, yes, it must be initialized as soon as possible, as you did.